robottelo/ui/base.py

```diff
@@ -370,8 +406,8 @@ def wait_until_element_is_clickable(
             )
             return None
 
-    def wait_until_element_is_not_visible(
-            self, locator, timeout=12, poll_frequency=0.5):
+    def wait_until_element_is_not_visible(self, locator, timeout=12,
+                                          poll_frequency=0.5):
```

Comment:
Is this necessary? It seems to go against the way we have been formatting functions' arguments throughout the code.